clients/python/Makefile

```diff
@@ -102,29 +95,41 @@ _check_venv_active:
 	# checking whether virtual environment was activated
 	@python3 -c "import sys; assert sys.base_prefix!=sys.prefix"
 
+
 .PHONY: install-dev
-install-dev: _check_venv_active ## install packages for development
-	pip install -r requirements/dev.txt
+install-dev: python-client _check_venv_active ## installs osparc_client and osparc in edit mode
+	python -m pip install -e artifacts/client
+	python -m pip install -e .
+	pip list
+
+
+#.PHONY: install-dev
+#install-dev: _check_venv_active ## install packages for development
+#	pip install -r requirements/dev.txt
 
 .PHONY: install-unit-test
 install-unit-test: _check_venv_active ## install packages for unit testing client
-	pip install -r $(PYTHON_DIR)/requirements/unit-test.txt
+	uv pip install -r $(CLIENTS_PYTHON_DIR)/requirements/unit-test.txt
 
 .PHONY: install-e2e-test
 install-e2e-test: _check_venv_active ## install packages for e2e testing client
-	pip install -r $(PYTHON_DIR)/requirements/e2e-test.txt
+	uv pip install -r $(CLIENTS_PYTHON_DIR)/requirements/e2e-test.txt
 
 .PHONY: install-doc
 install-doc: _check_venv_active install-dev ## install packages for generating documentation
-	pip install -r $(PYTHON_DIR)/requirements/doc.txt
+	uv pip install -r $(CLIENTS_PYTHON_DIR)/requirements/doc.txt
+
+
 
 .PHONY: pylint
 pylint: _check_venv_active ## runs linter (only to check errors. SEE .pylintrc enabled)
-	pylint --rcfile "$(PYTHON_DIR)/.pylintrc" -v "$(ARTIFACTS_DIR)/client"
+	pylint --rcfile "$(CLIENTS_PYTHON_DIR)/.pylintrc" -v src/osparc
 
 .PHONY: mypy
-mypy: $(SCRIPTS_DIR)/mypy.bash $(PYTHON_DIR)/mypy.ini ## runs mypy python static type-checker on this services's code. Use AFTER make install-*
-	@$(SCRIPTS_DIR)/mypy.bash client
+mypy: $(SCRIPTS_DIR)/mypy.bash $(CLIENTS_PYTHON_DIR)/mypy.ini ## runs mypy python static type-checker on this services's code. Use AFTER make install-*
```

Comment:
Did you check the mypy.ini for the bad issue?

Reply:
I am adding all the options in `pyproject.toml`